nixos/modules/services/x11/desktop-managers/plasma5.nix

```diff
@@ -455,8 +454,7 @@ in
       services.xserver.displayManager.defaultSession = mkDefault "plasma";
 
       environment.systemPackages =
-        with libsForQt5;
-        with plasma5; with kdeGear; with kdeFrameworks;
+        with plasma5; with kdeFrameworks;
```

Comment:
The second group of packages below all come from `kdeGear`. If we want to avoid overlapping `with`s, then we need to break up this list because `kdeGear` is still required.

Reply:
They all end up in libsForQt5/plasma5 because off https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/qt5-packages.nix#L62= which makes this entire list even more confusing.

Reply:
I couldn't run the tests on this pull request without adding `kdeGear` back. :shrug:

Reply:
Check ci sandro

Reply:
```suggestion
        with plasma5; with kdeGear; with kdeFrameworks;
```